src/Kunstmaan/MediaBundle/Helper/Imagine/BackgroundFilterLoader.php

```diff
@@ -9,6 +9,8 @@
 
 /**
  * This class can be removed when https://github.com/liip/LiipImagineBundle/issues/640 is fixed.
+ *
+ * @deprecated This class will be removed in MediaBundle 6.0 as the override is no longer necessary
```

Comment:
Can you also add a entry to the `UPGRADE-5.1` file?

Reply:
Done....